auditlog/models.py

```diff
@@ -452,8 +460,13 @@ def changes_display_dict(self):
                             )
                         else:
                             values_display.append(choices_dict.get(value, "None"))
-                    except Exception:
+                    except ValueError:
                         values_display.append(choices_dict.get(value, "None"))
+                    except Exception:
+                        if isinstance(value, list):
```

Comment:
Can we move this `isinstance` check to line 455 (where we test `if type(value) is list` rather than `if isinstance(value, list)`?